packages/alert/src/Alert.js

```diff
@@ -18,19 +18,21 @@ type Props = {
   /** Alert body */
   children?: any,
   closable?: boolean,
+  ariaLabelClose?: string,
   className?: string,
   onClose?: Function,
   /** Qa id */
   qa?: string,
 };
 
 class Alert extends Component<Props> {
+  state = {
+    closing: true,
+    closed: false
+  };
+
   constructor(props) {
```

Comment:
Shouldn't the constructor be removed if we use state like this?

Reply:
Yes, removed them everywhere where unnecessary.